extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/ranger/RuleAuthorization.scala

```diff
@@ -33,6 +33,7 @@ import org.apache.kyuubi.plugin.spark.authz.util.AuthZUtils._
 class RuleAuthorization(spark: SparkSession) extends Rule[LogicalPlan] {
   override def apply(plan: LogicalPlan): LogicalPlan = {
     plan match {
+      case subquery: Subquery => subquery
```

Comment:
So we mainly delay subquery to PrivilegeBuilder？

Reply:
> So we mainly delay subquery to PrivilegeBuilder？

Yea， let it checked by `buildSubquery()`, then we got a whole privilege objects for the whole plan.